src/apps/frontend/components/header/user-profile-snippet.component.tsx

```diff
@@ -9,7 +9,7 @@ import UserMenuDropdown from './user-menu-dropdown.component';
 import { UserMenuDropdownItem } from '.';
 
 interface DropdownUserProps {
-  account: Account;
+  account: Account | undefined;
```

Comment:
Not sure why would this be?

An account should always be present when dropdown is shown in header